src/access/RequestPath.h

```diff
@@ -26,8 +26,10 @@ namespace Access {
 struct RequestPath
 {
     // NOTE: eventually this will likely also contain node, for proxying
-    ClusterId cluster   = 0;
-    EndpointId endpoint = 0;
+    ClusterId cluster     = 0;
+    EndpointId endpoint   = 0;
+    uint32_t entityId     = 0; // attribute, command, or event id.  Ignored if entityWildcarded is true
```

Comment:
how about using a std::optional? then nullopt is "wildcard" otherwise an actual id.

How does one distinguish between this being a attribute, command or event ID? The description should explain this. The "or" is confusing. The IDs here are of a different namespace and are not unique.

E.g.  AccessControl::AccessControlExtensionChanged is event 1, AccessControl::extension is attribute 1,

Reply:
I think comment should be that this is based on MsgType ... however then why is this separated out? Maybe requestpath should instead be a structure of `request` that contains all information.

Reply:
I believe I have addressed these concerns.

Reply:
Resolving conversation, but tracking here for post 1.4 continuation: https://github.com/project-chip/connectedhomeip/issues/34913